memory_server.py

```
@@ -123,6 +123,22 @@ async def _run_review_in_background(lanlan_name: str):
         if lanlan_name in correction_cancel_flags:
             correction_cancel_flags[lanlan_name].clear()
 
+@app.post("/cache/{lanlan_name}")
+async def cache_conversation(request: HistoryRequest, lanlan_name: str):
+    """轻量级缓存：仅将新消息追加到 recent history，不触发 time_manager / review 等 LLM 操作。
```

Ruff (0.15.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)